WayPoints:                  
                            - {x: 0.0, y: 3.5, z: 2.8}
                            - {x: 0.0, y: 7.7, z: 2.8}
                            - {x: 2.75, y: 10.45, z: 2.8}
                            - {x: 4.9, y: 7.4, z: 2.8}
                            - {x: 4.9, y: 2.6, z: 2.8}
                            - {x: 2.6, y: 2.0, z: 1.2}
                            - {x: 2.6, y: 2.75, z: 1.2}
                            - {x: 2.0, y: 5.3, z: 1.2}
                            # - {x: 0.0, y: 0.0, z: 3.0}

MapTopic:                   '/gt_map'

OdomTopic:                  '/mavros/local_position/odom'

DilateRadius:               0.05

VoxelWidth:                 0.02

MapBound:                   [-25.0, 25.0, -25.0, 25.0, 0.0, 5.0]

TimeoutRRT:                 0.02

MaxVelMag:                  12.0

MaxBdrMag:                  2.1
# MaxBdrMag:                  3.84          # Iris

# MaxTiltAngle:               1.05
MaxTiltAngle:               0.785398      # Iris

MinThrust:                  2.0

# MaxThrust:                  12.0
MaxThrust:                  24.0          # Iris

# VehicleMass:                0.61
VehicleMass:                1.535         # Iris 1.535

GravAcc:                    9.8

HorizDrag:                  0.70

VertDrag:                   0.80

ParasDrag:                  0.01

SpeedEps:                   0.0001

WeightT:                    20.0

ChiVec:                     [1.0e+4, 1.0e+4, 1.0e+4, 1.0e+4, 1.0e+5]

SmoothingEps:               1.0e-2

IntegralIntervs:            16

RelCostTol:                 1.0e-5
